src/tests/testAccomod.spec.js

```diff
@@ -0,0 +1,177 @@
+      .set('Authorization', loggedtoken)
+      .field('address', 'kigali')
+      .field('name', 'hotel')
+      .end((err, res) => {
+        expect(res.status).to.equal(200);
+        expect(res.body).to.have.property('msg');
+        expect(res.body).to.have.property('status');
+        done();
+      });
+  });
+  it('It should return a 401 status, when a token is not provided', (done) => {
+    chai.request(myserver)
+      .patch('/api/v1/accommodations/1')
+      .send(mock[0])
+      .end((err, res) => {
+        expect(res.status).to.equal(401);
+        expect(res.body).to.have.property('msg');
+        expect(res.body).to.have.property('status');
+        expect(res.body.msg).to.equals('You are not authorized');
+        done();
+      });
+  });
+  it('It should return a 401 status, when a user is not a supplier', (done) => {
+    chai.request(myserver)
+      .patch('/api/v1/accommodations/1')
+      .set('Authorization', helper.createToken(2, 'alexis2@gmail.com', true, 'requestor'))
+      .send(mock[0])
+      .end((err, res) => {
+        expect(res.status).to.equal(401);
+        expect(res.body).to.have.property('msg');
+        expect(res.body).to.have.property('status');
+        done();
+      });
+  });
+  it('It should return a 400 status, when accommodation not found', (done) => {
+    chai.request(myserver)
+      .patch('/api/v1/accommodations/100')
+      .set('Authorization', loggedtoken)
+      .send(mock[0])
+      .end((err, res) => {
+        expect(res.status).to.equal(400);
+        expect(res.body).to.have.property('msg');
+        expect(res.body).to.have.property('status');
+        done();
+      });
+  });
+  it('It should return a 401 status, when accommodation does not belong to supplier', (done) => {
+    chai.request(myserver)
+      .patch('/api/v1/accommodations/1')
+      .set('Authorization', helper.createToken(2, 'alexis2@gmail.com', true, 'host'))
+      .send(mock[0])
+      .end((err, res) => {
+        expect(res.status).to.equal(401);
+        expect(res.body).to.have.property('msg');
+        expect(res.body).to.have.property('status');
+        done();
+      });
+  });
+  it('It should return a 400 status, when a wrong id is selected', (done) => {
+    chai.request(myserver)
+      .patch('/api/v1/accommodations/a')
+      .set('Authorization', loggedtoken)
+      .send(mock[0])
+      .end((err, res) => {
+        expect(res.status).to.equal(400);
+        expect(res.body).to.have.property('msg');
+        expect(res.body).to.have.property('status');
+        done();
+      });
+  });
+  it('It should return a 200 status, when an accommodation has been deleted', (done) => {
+    chai.request(myserver)
+      .delete('/api/v1/accommodations/1')
+      .set('Authorization', loggedtoken)
+      .end((err, res) => {
+        expect(res.status).to.equal(200);
+        expect(res.body).to.have.property('msg');
+        expect(res.body).to.have.property('status');
```

Comment:
Check that there is no accommodation with `id` of 1, after making this delete request